File: app/src/main/java/com/alexjlockwood/example/delight/ClockActivity.java
package com.alexjlockwood.example.delight;

import android.graphics.drawable.Animatable;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.widget.ImageView;

import butterknife.BindView;
import butterknife.ButterKnife;
import butterknife.OnClick;

public class ClockActivity extends AppCompatActivity {

  @BindView(R.id.alarm) ImageView alarmView;
  @BindView(R.id.clock) ImageView clockView;
  @BindView(R.id.timer) ImageView timerView;
  @BindView(R.id.stopwatch) ImageView stopWatchView;

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_clock);
    ButterKnife.bind(this);
  }

  @OnClick(R.id.rootview)
  void onClick() {
    ((Animatable) alarmView.getDrawable()).start();
    ((Animatable) clockView.getDrawable()).start();
    ((Animatable) timerView.getDrawable()).start();
    ((Animatable) stopWatchView.getDrawable()).start();
  }
}

File: app/src/main/java/com/alexjlockwood/example/delight/ClockActivity.kt
package com.alexjlockwood.example.delight

import android.graphics.drawable.Animatable
import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import android.widget.ImageView
import butterknife.BindView
import butterknife.ButterKnife
import butterknife.OnClick

class ClockActivity : AppCompatActivity() {
    @BindView(R.id.alarm)
    var alarmView: ImageView? = null

    @BindView(R.id.clock)
    var clockView: ImageView? = null

    @BindView(R.id.timer)
    var timerView: ImageView? = null

    @BindView(R.id.stopwatch)
    var stopWatchView: ImageView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_clock)
        ButterKnife.bind(this)
    }

    @OnClick(R.id.rootview)
    fun onClick() {
        (alarmView!!.drawable as Animatable).start()
        (clockView!!.drawable as Animatable).start()
        (timerView!!.drawable as Animatable).start()
        (stopWatchView!!.drawable as Animatable).start()
    }
}